src/main/java/org/rumbledb/expressions/Expression.java

```diff
@@ -46,11 +46,6 @@ protected Expression(ExceptionMetadata metadata) {
         super(metadata);
     }
 
-    @Override
```

Comment:
This implementation is not actually used; subclasses that do not override it have no children or should not be expressions.